project/src/test/java/com/google/sps/data/PlacesFetcherTest.java

```diff
@@ -124,17 +135,43 @@ public void fetch_zeroSearchResults_returnsEmptyList() throws Exception {
     PlacesFetcher spiedFetcher = spy(placesFetcher);
     doReturn(new PlacesSearchResult[0]).when(spiedFetcher)
       .getPlacesSearchResults(any(TextSearchRequest.class));
-    assertEquals(ImmutableList.of(), spiedFetcher.fetch());
+    assertEquals(ImmutableList.of(), spiedFetcher.fetch(PREFERENCES_BUILDER.build()));
   }
 
   @Test
   public void fetch_validSearchResults_returnsListOfPlaces() throws Exception {
     PlacesFetcher spiedFetcher = spy(placesFetcher);
-    doReturn(PLACE_DETAILS_1).doReturn(PLACE_DETAILS_2).when(spiedFetcher)
+    doReturn(PLACE_DETAILS_1).
```

Comment:
Formatting here should be:
Option 1: one line
```java
doReturn(foo).when(bar).getBaz();
```
Option 2:
```java
doReturn(foo)
  .doReturn(bar)
  .when(bar)
  .getBaz()
```

Reply:
(Same for other tests).

Reply:
Great thanks, I wondered where the line breaks should be. Fixed to option two.